index.html

```diff
@@ -0,0 +1,165 @@
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="./bootstrap-5.0.2-dist/css/bootstrap.min.css" rel="stylesheet">
+    
+    <link href="./bootstrap-5.0.2-dist/css/main.css" rel="stylesheet">
+    <link href="./bootstrap-5.0.2-dist/css/media-queries.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Globerry Couture</title>
+  </head>
+  <body>
+    
+    <section>
+        <div class="g-space-around">
+            <div class="text-end g-rightbg">
+                <ul>
+                    <li><a href=""><i class="fa fa-facebook"></i></a></li>
+                    <li><a href=""><i class="fa fa-twitter"></i></a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">My Page</a></li>
+                    <li><a href="">Register</a></li>
+                </ul>
+            </div>
+        </div>
+    </section>
+
+    <section class="container-fluid g-createbg">
+        <header class="container">
+           <nav class="navbar navbar-header navbar-expand-lg navbar-light">
+            <div class="container-fluid">
+              <a class="navbar-brand" href="index.html">
+                <img src="./images/globerry.PNG" class="top-img" alt="Globerry Logo"/>
+              </a>
+              <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+              </button>
+              <div class="collapse navbar-collapse justify-content-end" id="navbarNavAltMarkup">
+               
+                <div class="navbar-nav">
+                <a href="#" class="closeMenu"><i class="fa fa-times closeMenu" aria-hidden="true"></i></a>
+
+                  <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+                  <a class="nav-link" href="about.html">About Programme</a>
+                  <a class="nav-link" href="#">Designs</a>
+                  <a class="nav-link createborder" href="#">REGISTER NOW</a>
+                </div>
+              </div>
+            </div>
+          </nav>
+        </header>
+    </section>
+
+    <section class="contentbg">
+        <div class="container">
+            <h5>"Become a Fashionister"</h5>
+            <h1>REGISTER NOW AND<BR/>LEARN FASHION DESIGN<BR/>IN 2022</h1>
+            <p class="subtitle">Join our fashion session for the 2022 edition to learn how to empower yourself
+                and make the best dresses for your clients.
+            </p>
+            <div class="g-date">2022.10.15(THU) ~ 16(FRI)</div>
+            <div>@ 22 Olajide Agboola Street, Ago 40, Iyana Ipaja</div>
+        </div>
+    </section>
```

Comment:
- _(optional)_ Great job here. I only recommend giving the background image a blur or transparent property because from an accessibility point of view its not very easy the focus on the text
**imp**
![image](https://user-images.githubusercontent.com/37901469/169526262-fe517eee-8ef7-4de4-a002-327608549d7e.png)